ds701_book/24-NN-II-Backprop.qmd

```diff
@@ -367,6 +353,7 @@ In order to calculate the gradients, we will need to capture the computation gra
 To start with, we'll accept a tuple of child nodes in the initializer and store that as a set in the object.
 
 ```{python}
+#| code-fold: false
 # Value version 4
 class Value:
                         #    vvvvvvvvvvvv
```

Comment:
I think this is a typo.

Reply:
which? the vvvv's are pointing to the new argument, but I'm going to consolidate these to fewer versions anyway. stay tuned.